tasks/foot_traffic.py

```diff
@@ -0,0 +1,418 @@
+import os
+import urllib.request
+import math
+from luigi import Task, IntParameter, LocalTarget
+from lib.logger import get_logger
+from tasks.meta import current_session
+from tasks.util import (classpath, unqualified_task_id)
+from tasks.targets import PostgresTarget
+
+LOGGER = get_logger(__name__)
+
+
+def quadkey2tile(quadkey):
```

Comment:
https://github.com/CartoDB/python-quadkey

Reply:
I didn't know that... I'll change the code to use it. Thank you!

Reply:
at your command :)